cypress/e2e/shared/dashboardsView.test.ts

```diff
@@ -1392,4 +1392,268 @@ csv.from(csv: data) |> filter(fn: (r) => r.bucket == v.bucketsCSV)`
     cy.wait('@refreshCellQuery')
     cy.wait('@refreshSecondQuery')
   })
+
+  // auto refresh tests
+
+  describe('Dashboard auto refresh', () => {
+    let routeToReturnTo = ''
+    const jumpAheadTime = (timeAhead = '00:00:00') => {
+      return Cypress.moment()
+        .add(Cypress.moment.duration(timeAhead))
+        .format('YYYY-MM-DD HH:mm:ss')
+    }
+
+    beforeEach(() => {
+      cy.get('@org').then(({id: orgID, name}: Organization) => {
+        cy.createDashboard(orgID).then(({body}) => {
+          cy.fixture('routes').then(({orgs}) => {
+            routeToReturnTo = `${orgs}/${orgID}/dashboards/${body.id}`
+            cy.visit(routeToReturnTo)
+            cy.getByTestID('tree-nav')
+          })
+        })
+        cy.window().then(win => {
+          cy.wait(1000)
+          win.influx.set('newAutoRefresh', true)
+        })
+        cy.createBucket(orgID, name, 'schmucket')
+        const now = Date.now()
+        cy.writeData(
+          [
+            `test,container_name=cool dopeness=12 ${now - 1000}000000`,
+            `test,container_name=beans dopeness=18 ${now - 1200}000000`,
+            `test,container_name=cool dopeness=14 ${now - 1400}000000`,
+            `test,container_name=beans dopeness=10 ${now - 1600}000000`,
+          ],
+          'schmucket'
+        )
+      })
+      cy.getByTestID('button').click()
+      cy.getByTestID('switch-to-script-editor').should('be.visible')
+      cy.getByTestID('switch-to-script-editor').click()
+      cy.getByTestID('toolbar-tab').click()
+      const query1 = `from(bucket: "schmucket")
+  |> range(start: v.timeRangeStart, stop: v.timeRangeStop)
+  |> filter(fn: (r) => r["container_name"] == "cool")`
+      cy.getByTestID('flux-editor')
+        .should('be.visible')
+        .click()
+        .focused()
+        .type(query1)
+      cy.getByTestID('overlay').within(() => {
+        cy.getByTestID('page-title').click()
+        cy.getByTestID('renamable-page-title--input')
+          .clear()
+          .type('blah')
+        cy.getByTestID('save-cell--button').click()
```

Comment:
do we care that this is run in a beforeEach and not in one test? We're effectively testing that we can write and save some stuff in the overlay each time the tests below run

Reply:
This stuff is necessary for the setup for the tests